### DISCUSSION TITLE
Free text fix for Chinese characters

### DISCUSSION THREAD

**Discussion by ajthu233** [sumatrapdfreader/sumatrapdf#3095] [Q&A]
When insert free text of Chinese characters, some characters won't display. 
eg.  经济 铁轨 won't display in the box.

How to fix the problem?

**Comment by GitHubRulesOK** [comment]
SumatraPDF will add them
 but is unable to display using default western fonts in MuPDF render such as Helvetica/Times,
 If the characters are added ***or changed*** in any other editor to Arial then they become visible ! (Edge may have similar problem ?)

![image](https://user-images.githubusercontent.com/12754913/201567189-52074d12-9371-479f-9330-357fdfc03465.png)


![image](https://user-images.githubusercontent.com/12754913/201567780-5632ad8c-302d-43d9-a475-dae4118d577a.png)
**After save in Edge**
![image](https://user-images.githubusercontent.com/12754913/201567819-08296043-2787-4fb2-9197-1445fbda98f2.png)
